src/main/java/duke/task/TypeOfTask.java

```diff
@@ -0,0 +1,20 @@
+package duke.task;
+
+/**
+ * Enum that contains the various types of tasks accepted by Duke
+ * e.g.,<code>todo</code>, <code>deadline</code>, <code>event</code>
+ */
+public enum TypeOfTask {
```

Comment:
This isn't really a suggestion but just a question! I'm just curious to know why you created an enum class for the different types of task if there are already classes for the different types of task which inherit the Task class.